lcls-twincat-pmps/PMPS/GVLs/PMPS_GVL.TcGVL

```diff
@@ -62,6 +75,7 @@ VAR_GLOBAL CONSTANT
         nTran := 1,
         neVRange :=  2#1111_1111_1111_1111_1111_1111_1111_1111,
         nRate    := UDINT#1000000,
+		nBCRange := 2#0111_1111_1111_1111,
```

Comment:
If the "spare" bps end up being bad, we can switch this to `2#0001_1111_1111_1111`

Reply:
spare bps are ok so I can mark this comment as resolved